src/autora/experimentalist/leverage/__init__.py

```diff
@@ -69,6 +77,7 @@ def sample(
             sd: A noise parameter around the selected samples to allow for the selection
                 of datapoints that are not part of the original dataset.
                 This is not currently constrained by the pipelines IV resolution.
+            random_state:
```

Comment:
@younesStrittmatter shall we add the description to this `arg`? eg, “ the seed value for the random number generator”